webrtc/libjingle/xmpp/boshsocket.cc

```diff
@@ -155,7 +156,15 @@ namespace buzz {
 #ifdef FEATURE_ENABLE_SSL
     if (tls_ != buzz::TLS_DISABLED) {
       first_socket = rtc::SSLAdapter::Create(first_socket);
+// For non-production builds, relax certificate requirements. This will allow the use
+// of a debugging proxy, for example.
+#ifndef _PRODUCT
```

Comment:
Ah. You think we should do that along with this PR?

And would you do something like setting of PSY_PRODUCT=1 for non-Windows
builds (keeping PSY_PRODUCT for Windows at discretion of builder)?


On Fri, Sep 18, 2015 at 1:33 PM, Will Kelleher <notifications@github.com>
wrote:

> In webrtc/libjingle/xmpp/boshsocket.cc
> <https://github.com/PersonifyInc/webrtc/pull/13#discussion_r39886539>:
>
> > @@ -155,7 +156,15 @@ namespace buzz {
> >  #ifdef FEATURE_ENABLE_SSL
> >      if (tls_ != buzz::TLS_DISABLED) {
> >        first_socket = rtc::SSLAdapter::Create(first_socket);
> > +// For non-production builds, relax certificate requirements. This will allow the use
> > +// of a debugging proxy, for example.
> > +#ifndef _PRODUCT
>
> I meant that we probably aren't compiling WebRTC with PSY_PRODUCT set,
> today, but we can change that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PersonifyInc/webrtc/pull/13/files#r39886539>.
>